webcompat/templates/issue/issue-header.html

```diff
@@ -0,0 +1,20 @@
+{% from "web_modules/tag.html" import tag with context %}
+
+<header class="wc-IssueDetail">
+  <h1 class="wc-IssueDetail-title">
+    <script type="text/template" id="title-tmpl">
+    Issue <%= number %>: <%- title %>
+    </script>
+  </h1>
+  <!-- Issue Date -->
+  <div class="wc-IssueDetail-create">
+    <script type="text/template" id="metadata-tmpl">
+    <div class="wc-Tag wc-Tag--label wc-Tag--<%= stateClass %>"><%= issueState %></div>
```

Comment:
@miketaylr do you know how I can use macro  like this: `{{ tag('filter', <%= stateClass %>) }}` 

If I do this, I have an error.. 

poke @karlcow